src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
In reply to Fred's question on "in VIP how is distributed the initial number of particles among the workers?", the short answer is: "for the static workflow, each node computes the same fraction of the work". The long answer is that the fraction is computed either on the number of particles, or on the GATE time runs (which are very different of the real time the execution takes). Moreover, this is not decided at the workflow level (VIPServer), but at the worker level, based on the parsing of the GATE macro file. If we want to go to this level of details, there will be a few things we need to add to the simulator and the log parser (to detect whether the splitting was done on time or events). 

So I do believe that the details of the splitting algorithm can be left aside for a while, as long as we manage to reproduce the functioning by injecting the execution time.

Yes, we will stop injecting the execution time at some point, but this is a long-term goal. In my opinion, we should handle the splitting at the time we will really need it. It is not about "patching up things later", is about adding elements step by step.